roles/installer/ospd/overcloud/tagging/tasks/main.yml

```diff
@@ -33,3 +33,12 @@
   with_nested:
       - "{{ node_list.results }}"
       - "{{ tagged_flavors }}"
+
```

Comment:
why can't this be part of the role for hostnames? (as preparation)
I would prefer not to "pollute" a general playbook with "task specific" stuff.